requirements.txt

```diff
@@ -2,4 +2,5 @@ mavsdk>=1.4.0
 numpy>=1.23.0
 pymavlink>=2.4.37
 grpcio>=1.47.0
-pyserial>=3.5
\ No newline at end of file
+pyserial>=3.5
+pyulog
```

Comment:
Where do we use this?

Reply:
The pyulog is what we need for Ulog to CSV conversions. I forgot about pyserial.